analyzers/src/SonarAnalyzer.CFG/Helpers/RoslynHelper.cs

```diff
@@ -22,6 +22,7 @@ namespace SonarAnalyzer.CFG.Helpers
 {
     public static class RoslynHelper
     {
+        public const int UnsupportedVersionVS2017 = 2;
```

Comment:
- I would rename this to `VS2017MajorVersion`, I think the code will still be easy to understand.
- My understanding is that VS2015 is unsupported and VS2017 is deprecated.